### PR TITLE
=NO-TICKET: Change table structure used as input

### PR DESCRIPTION
**Pull request by Bettiher**
Tiny change in Pseudocone in order to use the new structure in the datasets created by the garden > `anon_activitytime` column instead of `activity_time`.
I adapted the README with relevant examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,7 @@ The function of the Pseudocone component is to fetch data that is used for offli
  1. to deliver a list of test users to Justicia (together with items they have consumed); and
  2. to deliver user history items to bramble to be used to generate recommendations (copies the behaviour of
  [Bristlecone](https://github.com/bbc/connected-data-bristlecone))
- instead of reading user actions from UAS it reads from an offline UAS dump JSON file (named
- "anonymised_uas_extract.json") which is located in a GCP bucket named "pseudocone_data_dump".
+ instead of reading user actions from UAS it reads from offline UAS JSON dumps "end_YYYY_MM_DD_window_30_resource_types_clip_activity_types_plays_limit_none.json" located in the anonymised_logs folder in the GCP bucket "pseudocone_data_dump" in the bbc-datalab project, see [here](https://console.cloud.google.com/storage/browser/pseudocone_data_dump_v2/anonymised_logs/?project=bbc-datalab&folder=true&organizationId=33845241025&showFTMessage=false).
```
Comment (maria-panteli) [inline README.md]: pseudocone_data_dump_v2

File: README.md [hunk 2]
```diff
@@ -69,24 +64,23 @@ This should return an empty response.
...
+grpcurl -protoset ./pseudocone.protoset -plaintext -d '{"user": {"id": "6fcba5ec5f0d00efa0a9ed3a5053a20f3bdc4fdc127497a7556a046ca61bd50e"}, "dataset":"anonymised_logs/users_logs_sample.json", "end_interaction_time":"2019-01-15T00:02:10.5Z"}' localhost:50057 pseudocone.PseudoconeService.ListInteractions```
 ```
 
 5. ListTestDataUsersBetweenDates
-
 ```
-grpcurl -protoset ./pseudocone.protoset -plaintext -d '{"limit":8, "offset":1, "users":[{"id":"1155"}, {"id": "4535"}], "start_interaction_time": "2018-01-01T00:00:26.318497Z", "end_interaction_time":"2018-05-01T00:00:26.318497Z", "dataset":"anonymised_uas_extract_list.json"}' localhost:50057 pseudocone.PseudoconeService.ListTestDataUsersBetweenDates
+grpcurl -protoset ./pseudocone.protoset -plaintext -d '{"limit":8, "offset":1, "users":[{"id":"13f13c1631d66a5948a29f1f483c8e7f30f1b71b389cecf10eb19072bda39682"}, {"id": "6fcba5ec5f0d00efa0a9ed3a5053a20f3bdc4fdc127497a7556a046ca61bd50e"}], "start_interaction_time": "2019-01-01T00:00:26.318497Z", "end_interaction_time":"2019-01-12T21:28:18.318497Z", "dataset":"anonymised_logs/users_logs_sample.json"}' localhost:50057 pseudocone.PseudoconeService.ListTestDataUsersBetweenDates
```
Comment (maria-panteli) [inline README.md]: end_interaction_time on `2019-01-15` for consistency with previous command

**Review by maria-panteli — APPROVED** [review]
looks good!

**Comment by douglaspenna** [comment]
Looks good to me! 🚀

**Review by douglaspenna — APPROVED** [review]
:shipit:

**Pull request merged**